docs/README.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to LocalStack
+
+We welcome contributions to LocalStack, under the [Contributor LicenseAgreement (CLA)][1].
+
+The guides in this section are for developers of LocalStack, to better understand how LocalStack works internally, how to set up local development environments, and how to contribute to the codebase.
+
+  [1]: https://github.com/localstack/localstack/blob/master/.github/CLA.md
+
+## Sections
+
+- [LocalStack Concepts](Concepts/index.md)
+- [Development Environment Setup](development-environment-setup.md)
```

Comment:
I think the order should be changed here a little bit, and the first link should be our contribution guidelines:
```suggestion
- [Contribution Guidelines](contributing.md)
- [Development Environment Setup](development-environment-setup.md)
- [LocalStack Concepts](Concepts/index.md)
```